playbooks/viya-upgrade/pre_upgrade/verify-postgres-server.yml

```diff
@@ -76,7 +79,7 @@
             msg: "{{ postgres_cluster_unhealthy }}"
           register: assert_postgres_nodes
           when: pool_cdf.stat.exists
-      when: postgres11_plus_found.stat.exists | bool == false
+      when: postgres11_plus_found | bool == false
```

Comment:
Since the `postgres11_plus_found` fact is now a `bool`, I think these `postgres11_plus_found | bool == false` could be simplified to `not postgres11_plus_found` (but it's been a while since I looked at Ansible.

Reply:
I'll do a quick local test

Reply:
you're right, i'll make the change real quick